### DISCUSSION TITLE
IPAddress is not highlighted

### DISCUSSION THREAD

**Discussion by JokerGTS** [espressif/arduino-esp32#6729] [Q&A] [answered]
When I used version 1.0.5 for Arduino IDE, the IPAddress was highlighted, but when I upgraded to version 2.0.3, the IPAddress stopped highlighting. Is that how it should be?
Before update:
![image](https://user-images.githubusercontent.com/48904926/167923523-18ea2499-e4a1-4474-8e57-a9936498f2b5.png)
After update:
![image](https://user-images.githubusercontent.com/48904926/167923419-c825608d-b9ec-4273-a353-8be6309570ae.png)

### UPD:
I added the `IPAddress	KEYWORD1` line to the Datatypes section in `AppData\Local\Arduino15\packages\esp32\hardware\esp32\2.0.3\libraries\WiFi\keywords.txt ` file and it worked, please fix it.
![3234](https://user-images.githubusercontent.com/48904926/168045223-c6a91bac-d543-4e22-bec1-76a2b0570e75.jpg)

**Comment by me-no-dev** [comment, marked as answer]
IPAddress is part of official Arduino API (and supported in the IDE). Why it's not colored anymore is unknown. We surely did not change anything to cause it.

**Reply by JokerGTS** [reply]
Could you tell me if you have highlight in version 2.0.3?

**Reply by JokerGTS** [reply]
When I select another board package (eg ESP8266), the highlight works.
